core/templatetags/discover_uni_tags.py

```diff
@@ -39,3 +39,8 @@ def create_list(*args):
 def insert_values_to_rich_text(*_, **kwargs):
     list(kwargs.get('substitutions'))
     return kwargs.get('content').source % (kwargs.get('substitutions'))
+
+
+@register.simple_tag
+def length_of_list(view_list):
+    return len(view_list)
```

Comment:
Very minor point Matt:

The name implies it's a generic length of list function, but the param suggests it's only ever used for view_list.

If only ever used for view list length maybe better name length_of_view_list(view_list)

If could be used for other things, maybe just a generic length_of_list(list) i.e., list rather than view_list as the param.

Reply:
It is a generic length of list function.

The reason I didn't call the param list is because pycharm was complain it mirrored the name of the built in python list function.
I went with view_list because its a template tag and any list it gets comes from the view/template.
Happy to change it to something else. I don't know if mirroring the list function is considered bad practice?

Reply:
Of course Matt, silly me you can't use list - perhaps list_object or python_list would be more generic and not clash with the builtin type.  However if the list is always a list provided by a view then view_list seems fine to me. Anyway, I don't want to waste your time on this Matt it's not a big deal.